docs/configuration.md

```diff
@@ -1010,6 +1010,19 @@ Apart from these, the following properties are also available, and may be useful
   </td>
   <td>2.2.1</td>
 </tr>
+<tr>
+  <td><code>spark.shuffle.accurateBlockSkewedFactor</code></td>
```

Comment:
I see that the `spark.shuffle.accurateBlockThreshold` configuration is already documented in this table. It looks like we're preexistingly inconsistent about alphabetizing this list.

What do you think about either moving this new configuration a bit further down so it's next to `spark.shuffle.accurateBlockThreshold`?

Reply:
Done, also updated the test section
<img width="1274" alt="image" src="https://github.com/user-attachments/assets/f202fcd0-176a-4b5f-9901-017167fbaf65">